docs/release-notes/8.0.asciidoc

```diff
@@ -8,3 +8,35 @@
 [discrete]
 [[known-issues-8.0.0-rc2]]
 ==== Known issues
```

Comment:
Do we have (or want to have) some standardized wording that says that we plan to fix these issues in a future release?

Reply:
Not that I'm aware of. I'm open to adding that language in if requested though!